droughtpipeline/forecast.py

```diff
@@ -182,9 +182,9 @@ def __compute_triggers(self,debug: bool = False, datestart: datetime = date.toda
         if debug:
             scenario = os.getenv("SCENARIO", "Forecast")  # TODO: pull scenario debug to a proper scenario script
             if scenario == "Warning":
-                trigger_on_minimum_probability = 0.1
+                trigger_on_minimum_probability = 0.3
```

Comment:
@p-phung shouldn't this be 0.01 as well?

Reply:
I was confused with the one in extract.py where there is a `Trigger` with 0.01 and a `Warning` with 0.3.

What are the exact scenarios we have here? `Warning` and `NoWarning` only?